src/components/MovieCard.tsx

```diff
@@ -0,0 +1,15 @@
+import { IMG_CDN_URL } from "../utils/constants";
+
+interface PosterPathProps {
```

Comment:
Can use type instead of interface as it fulls the above requirement and not inheriting anything or extending it

Reply:
resolved